tests/test_create.py

```diff
@@ -106,14 +106,16 @@ def test_pipeline_creation_with_yml_skip(self, tmp_path):
         pipeline.init_pipeline()
 
         # Check pipeline template yml has been dumped to `.nf-core.yml` and matches input
-        assert not os.path.exists(os.path.join(pipeline.outdir, "pipeline_template.yml"))
-        assert os.path.exists(os.path.join(pipeline.outdir, ".nf-core.yml"))
-        with open(os.path.join(pipeline.outdir, ".nf-core.yml")) as fh:
+        assert not os.path.exists(Path(pipeline.outdir / "pipeline_template.yml"))
+        assert os.path.exists(Path(pipeline.outdir / ".nf-core.yml"))
+        with open(Path(pipeline.outdir / ".nf-core.yml")) as fh:
             nfcore_yml = yaml.safe_load(fh)
             assert "template" in nfcore_yml
             assert yaml.safe_load(PIPELINE_TEMPLATE_YML.read_text()).items() <= nfcore_yml["template"].items()
 
         # Check that some of the skipped files are not present
-        assert not os.path.exists(os.path.join(pipeline.outdir, "CODE_OF_CONDUCT.md"))
-        assert not os.path.exists(os.path.join(pipeline.outdir, ".github"))
-        assert not os.path.exists(os.path.join(pipeline.outdir, "conf", "igenomes.config"))
+        assert not os.path.exists(Path(pipeline.outdir / "CODE_OF_CONDUCT.md"))
+        assert not os.path.exists(Path(pipeline.outdir / ".github"))
+        assert not os.path.exists(Path(pipeline.outdir / "conf" / "igenomes.config"))
+        assert not os.path.exists(Path(pipeline.outdir / ".editorconfig"))
```

Comment:
```suggestion
        assert not os.path.exists(pipeline.outdir / "CODE_OF_CONDUCT.md")
        assert not os.path.exists(pipeline.outdir / ".github")
        assert not os.path.exists(pipeline.outdir / "conf" / "igenomes.config")
        assert not os.path.exists(pipeline.outdir / ".editorconfig")
```

Reply:
why are you remove path but not os.path.exists? 😁

Reply:
I overlooked the `os.path.exists` 😂